src/components/atoms/Slider/index.tsx

```diff
@@ -34,14 +34,19 @@ export const Slider = ({ swiperRef }: SliderProps) => {
         }}
         ref={swiperRef}
       >
-        {completedProfileMentors?.map(mentor => {
+  { mentors.map((mentor) => {
```

Comment:
Sugestão de melhoria:
Para evitar erros quando `mentors` não estiver presente ou não for um array, você pode usar uma das seguintes abordagens:

1. Verificar explicitamente se `mentors` existe e é um array antes de mapear:
{mentors && Array.isArray(mentors) && mentors.map((mentor) => (
  <div key={mentor.id}>
    {/* Renderize o conteúdo do mentor aqui */}
  </div>
))}

2. Usar encadeamento opcional (optional chaining) para verificar se `mentors` existe:
{mentors?.map((mentor) => (
  <div key={mentor.id}>
    {/* Renderize o conteúdo do mentor aqui */}
  </div>
))}

A primeira abordagem garante que `mentors` é um array antes de tentar mapear, o que é uma verificação mais robusta.
A segunda abordagem é mais concisa e moderna, mas assume que `mentors` será um array se existir.
Escolha a abordagem que melhor se adequa ao seu caso de uso.